python/taichi/ui/window.py

```diff
@@ -26,14 +26,14 @@ class Window:
         vsync (bool): whether or not vertical sync should be enabled.
         show_window (bool): where or not display the window after initialization.
     """
-    def __init__(self, name, res, vsync=False, show_window=True):
+    def __init__(self, name, res, pos=(0, 0), vsync=False, show_window=True):
```

Comment:
pos=(0,0) might be a problem for Windows users who put the task bar on top. Consider (100, 100) or something like that.

Reply:
Let me try try on Windows.

Reply:
You are right, I will fix that.

Reply:
If you need to add a kwarg, it'd be better to add it at the end to avoid breaking current api. e.g.  if user had `Window('test', (512, 512), False, True)` it'll be broken by this PR. 
Also `pos` need a proper docstring explaining which window, and where is the origin ;)